src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/test/WattTimeDataSourceTests.cs

```diff
@@ -48,7 +48,7 @@ public void Setup()
         this.DataSource = new WattTimeDataSource(this.Logger.Object, this.WattTimeClient.Object, this.LocationSource.Object);
 
         this.DefaultLocation = new Location() { Name = "eastus" };
-        this.DefaultRegion = new RegionResponse() { Region = "TEST_REGION", RegionFullName = "Test Region Full Name", SignalType = SignalTypes.co2_moer };
+        this.DefaultRegion = new RegionResponse() { Region = "TEST_REGION", RegionFullName = "Test Region Full Name", SignalType = "co2_moer" };
```

Comment:
same for this file too, leverage the enums to strings, not string literals please.